help/db_meeting-data.md

```diff
@@ -0,0 +1,47 @@
+### Zawartość tabel
+
+##### Dictionary -- nowe słowniki
```

Comment:
Że zaczynasz od h3 to jakoś do przeżycia, ale że potem przeskakujesz na h5 to już nie wiem czemu. Jak masz kiepski renderer do tych stron to se weź lepszy albo co.